README.md

```diff
@@ -0,0 +1,35 @@
+# My portfolio project
+This is a porfolio project that has my details and infromation. 
```

Comment:
- [ ] Kindly make sure the description of your project goes here. You can see the [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Estimated%20time%3A%202.5h-,Description,-For%20the%20first)